src/maelstrom/workload/lock.clj

```diff
@@ -0,0 +1,79 @@
+(ns maelstrom.workload.lock
+  "A distributed lock workload: sends lock and unlock messages."
+  (:require [maelstrom [client :as c]]
+            [jepsen [checker :as checker]
+                    [client :as client]
+                    [generator :as gen]]
+            [schema.core :as s]))
+
+(c/defrpc lock!
+  "Clients send `lock` messages to servers. Servers should respond with
+  `lock_ok` messages when the lock is granted."
+  {:type (s/eq "lock")}
+  {:type (s/eq "lock_ok")})
+
+(c/defrpc unlock!
+  "Clients send `unlock` messages to servers. Servers should respond with
+  `unlock_ok` messages acking that the lock is released"
```

Comment:
What happens if the lock is not currently held? What happens if the lock is held by a different client?

Reply:
It will cause the check to fail.

Reply:
Same deal here--there should be descriptions of these semantics either in these docstrings or in the ns docs.